org.okstar.ok-msg.yml

```diff
@@ -0,0 +1,101 @@
+id: org.okstar.ok-msg
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: ok-msg-desktop
+rename-icon: ok-msg
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --device=all
+  - --talk-name=org.gnome.Mutter.IdleMonitor
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=com.canonical.indicator.application
+  - --filesystem=home:rw
+
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    add-ld-path: .
+    version: '23.08'
+    autodownload: true
+    autodelete: false
+  org.freedesktop.Platform.GL.default:
+    version: '23.08'
+    autodownload: true
+    autodelete: false
+
+build-options:
+  strip: false
+  no-debuginfo: true
+  build-args:
+    - --share=network
+cleanup:
+  - /doc
+  - /include
+  - /lib/cmake
+  - /lib/libexec
+  - /lib/metatypes
+  - /lib/pkgconfig
+  - /lib/systemd
+  - /mkspecs
+  - /modules
+  - /share/doc
+  - /share/man
+  - '*.prl'
+  - '*.la'
+  - '*.a'
+
+modules:
+  - name: tcl
+    buildsystem: simple
+    build-commands:
+      - cp -r ../tcl ${FLATPAK_DEST}/
+      - cd ${FLATPAK_DEST}/tcl/unix; ./configure --prefix=${FLATPAK_DEST}/ --exec-prefix=/app && make && make install
```

Comment:
you haven't done what I suggested. It is still the same thing as before.

`${FLATPAK_DEST}` is NOT a build directory.